### PR TITLE
An example how to call JS from Rust using Deno executable

### PR DESCRIPTION
**Pull request by sergey-shandar**
First part of #14

Doc preview: https://github.com/Trust-Machines/core-eng/blob/sergey/js/README.md

### CODE REVIEW

**Review by netrome — APPROVED** [review]
lgtm. Is the idea to later support arbitrary code evaluation by using something like [eval()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval)?

**Comment by sergey-shandar** [comment]
> lgtm. Is the idea to later support arbitrary code evaluation by using something like [eval()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval)?

I was thinking more about like [RPC](https://en.wikipedia.org/wiki/Remote_procedure_call). However, some commands may execute JS code using [Function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/Function), `eval` = `evil`.

**Comment by sergey-shandar** [comment]
See also https://github.com/hirosystems/clarinet/ for interop.

**Comment by sergey-shandar** [comment]
@netrome, I've introduced some changes in the PR since you reviewed the code. In particular, the protocol message now is `${JSON}\n". It allows ti use `demo test.mjs` as a console.

**Comment by netrome** [comment]
> @netrome, I've introduced some changes in the PR since you reviewed the code. In particular, the protocol message now is `${JSON}\n`. It allows to use `demo test.mjs` as a console.

Cool, I'll re-review later today

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -16,6 +16,9 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+    - uses: denoland/setup-deno@v1
+      with:
+        deno-version: v1.x
```
Comment (netrome) [inline .github/workflows/rust.yml:21]: I'd prefer to specify a very specific version to make sure we don't have any CI time bombs.

**Review by netrome — APPROVED** [review]
Just one minor opinion.

**Review by igorsyl — APPROVED** [review]
lgtm! small and effective!

**Pull request merged**